server/context.go

```diff
@@ -165,12 +166,12 @@ func (ctx *Context) initEngine(attrsStr, path string) (engine.Engine, error) {
 	return engine.NewRocksDB(attrs, path, ctx.CacheSize), nil
 }
 
-// parseAttributes parses a colon-separated list of strings,
+// parseAttributes parses a list of strings separated by sep,
 // filtering empty strings (i.e. ",," will yield no attributes.
 // Returns the list of strings as Attributes.
-func parseAttributes(attrsStr string) proto.Attributes {
+func parseAttributes(attrsStr string, sep string) proto.Attributes {
```

Comment:
So then it sounds to me like the node-level attributes should be parsed with `:` instead of `,`. `--store=hdd:7200rpm=/mnt/hda1,ssd=/mnt/ssd1 --attrs=us-east:gpu` (so no code change, just update the docs of --attrs)